packages/cli/src/cmds/validator/options.ts

```diff
@@ -61,6 +61,7 @@ export type IValidatorCliArgs = AccountValidatorArgs &
     "http.responseWireFormat"?: string;
 
     "externalSigner.url"?: string;
+    "externalSigner.urls"?: string[]; 
```

Comment:
we shouldn't add a new option, can change previous one and add `externalSigner.url` as an alias at the cli level